packages/cli-kit/src/private/node/ui/components/SelectPrompt.tsx

```diff
@@ -132,7 +143,16 @@ function SelectPrompt<T>({
       ) : null}
 
       {gitDiff && !submitted ? (
-        <Box marginLeft={7} marginTop={1}>
+        <Box
```

Comment:
Why is this part in a separate box? Could we put the `gitDiff` inside the part above right below infoTable? Like:

```
{(infoTable || infoMessage || gitDiff) && !submitted ? (
        <Box
          marginTop={1}
          marginLeft={3}
          paddingLeft={2}
          borderStyle="bold"
          borderLeft
          borderRight={false}
          borderTop={false}
          borderBottom={false}
          flexDirection="column"
          gap={1}
        >
          {infoMessage ? (
            <Box flexDirection="column" gap={1}>
              <Text color={infoMessage.title.color}>
                <TokenizedText item={infoMessage.title.text} />
              </Text>
              <TokenizedText item={infoMessage.body} />
            </Box>
          ) : null}
          {infoTable ? <InfoTable table={infoTable} /> : null}
          {gitDiff ? <GitDiff {...gitDiff} /> : null}
        </Box>
      ) : null}
```

Reply:
Ah sorry this suggestion is already in main, if you solve the conflicts with this branch we'll get this for free.